.pipelines/azure-pipelines.yml

```diff
@@ -24,7 +24,6 @@ variables:
   # This is used in functional tests to make certain tests mandatory
   # in CI, but optional during local development. 
   SCITT_CI: 1
-
   ACR_REGISTRY: scittoss.azurecr.io
```

Comment:
Any reason to maintain a different registry and not push to the existing `confidentialledgeracrstaging` ACR? Just trying to understand if it's worth maintaining a different ACR at this point if we are not planning to onboard to MAR.

Reply:
My preference is to separate SCITT entirely to be honest. It's easy to replace it any time in future if needed, the biggest issue was configuring the pools and Az pipelines on ADO

Reply:
Sounds good. However, keep in mind this means we need to configure required permissions for our AKS clusters to pull images from the new registry and modify the scitt helm chart accordingly to pull the CCF app container image from the new registry.

Reply:
This is a valid point but let's discuss more offline, I think that depends on some ongoing discussion about the distribution of the service